domehunter/dome_control.py

```diff
@@ -279,6 +289,11 @@ def dome_in_motion(self):
         logger.debug(f'Dome in motion: {dome_motion}')
         return dome_motion
 
+    @property
+    def movement_thread_active(self):
+        """Return true if a movement thread is running"""
+        return self._move_event.is_set()
```

Comment:
I would also suggest a more intuitive name that doesn't include `thread`. Depends on what logic you want, but maybe `is_moving`.

Reply:
Might be right, we also already have a `.dome_in_motion` property that is based off the rotation_relay state. I'm no longer sure if we even need this threading event.